packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -448,7 +448,12 @@ class ControlledMultiSelectExample extends React.Component {
 
 ### Multi select with search filter
 
-When there are many options, you could use a search filter in the MultiSelect. The search filter will be performed toward the labels of the option items.
+When there are many options, you could use a search filter in the MultiSelect.
+The search filter will be performed toward the labels of the option items.
+
+*NOTE:* If there are more than 100 options, the component will automatically use
```

Comment:
I'd frame this as "If there are 100+ options, ...`

Reply:
Changed the copy here to point out we are using `react-window` for all the cases.